nginx-pckg-module/src/media/mpegts/adts_encoder_filter.c

```diff
@@ -42,6 +42,16 @@ adts_encoder_set_media_info(
     adts_frame_header_set_channel_configuration(state->header, codec_config->channel_config);
     adts_frame_header_set_adts_buffer_fullness(state->header, 0x7ff);
 
+#if (VOD_DEBUG)
+
+    ngx_log_error(NGX_LOG_INFO, context->request_context->log, 0,
+            "adts_encoder_set_media_info: mp4a object_type: %D sample_rate_index: %D channel_config: %D",
+            codec_config->object_type,
+            codec_config->sample_rate_index,
+            codec_config->channel_config);
+
```

Comment:
instead of if + log_error, just use log_debug -
```
vod_log_debug3(VOD_LOG_DEBUG_LEVEL, context->request_context->log, 0,
    "adts_encoder_set_media_info: mp4a object_type: %D sample_rate_index: %D channel_config: %D",
     codec_config->object_type,
     codec_config->sample_rate_index,
     codec_config->channel_config);
```

Reply:
no, i want it to be info